.github/workflows/release.yml

```diff
@@ -43,14 +45,16 @@ jobs:
       - name: Package release
         id: package_release
         run: |
-          VERSION=`cat semver.txt`
+          dotnet tool restore
+          VERSION=`nbgv get-version --variable Version`
+          NUGETVERSION=`nbgv get-version --variable NuGetPackageVersion`
```

Comment:
There seems to be causing a problem here. From my tests it looks that Version return a 4 segment version e.g. 3.6.0.2 while nugetversion returns a SemVer version.

Reply:
> Aside from that, it automatically inferred, that the next version is 3.0.6. I guess that is correct behaviour.

Yes, the version for the next version shall be configured after a release using **nbgv tool** or **manually**.

<img width="1011" alt="image" src="https://github.com/CycloneDX/cyclonedx-dotnet/assets/2315215/95ee35e7-e3d7-4f67-8fa1-dce142ec8620">